pywr-schema/src/nodes/virtual_storage.rs

```diff
@@ -39,6 +39,15 @@ pub struct AnnualReset {
     pub month: u8,
 }
 
+#[derive(serde::Deserialize, serde::Serialize, Clone, Debug, JsonSchema, PywrVisitAll)]
+#[serde(deny_unknown_fields)]
+pub struct SeasonalReset {
```

Comment:
It would be good to have the docstring of the virtual storage update to mention this reset option